omnigibson/transition_rules.py

```diff
@@ -153,6 +153,10 @@ def step(cls):
                 added_obj_attrs += output.add
                 removed_objs += output.remove
 
+        cls.execute_transition(added_obj_attrs=added_obj_attrs, removed_objs=removed_objs)
```

Comment:
For reviewer context: this is a simple refactor to allow triggering of rules by symbolic primitives.